src/types/CompareCertResp.ts

```diff
@@ -29,5 +30,10 @@ export const deserializeCompareCertResp = (stream: VectorBufferStream): CompareC
 
   const obj: CompareCertRespSerializable = Utils.safeJsonParse(stream.readString())
 
+  const errors = verifyPayload('CompareCertResp', obj)
```

Comment:
Why isnt `AJVSchemaEnum...` being used here like it is in ApoptosisProposalReq.ts?

Reply:
Fixed